packages/web-app-files/src/composables/sort/useSort.ts

```diff
@@ -64,47 +74,52 @@ export function useSort<T>(options: SortOptions<T>): SortResult<T> {
   }
 }
 
-function createSortByQueryRef<T>(options: SortOptions<T>): Ref {
+function createSortByQueryRef<T>(options: SortOptions<T>): Ref<QueryValue> {
   if (options.sortBy) {
     return isRef(options.sortBy) ? options.sortBy : ref(options.sortBy)
   }
 
   return useRouteQueryPersisted({
     name: unref(options.sortByQueryName) || SortConstants.sortByQueryName,
-    defaultValue: firstSortableField(unref(options.fields))?.sortBy,
```

Comment:
I believe this can never have worked, as `firstSortableField` used to be a string. 
This fix required the change required adding the router.replace mock.